.travis.yml

```diff
@@ -1,16 +1,17 @@
 language: minimal
 
-dist: xenial
+dist: bionic
 
 addons:
   apt:
     packages:
-      - docker-ce
       - python3-pip
       - python3-setuptools
       - clang-format
 
 install:
+  # Install the most recent version of Docker to use DOCKER_BUILDKIT.
+  - bash scripts/travis/install-docker.sh
```

Comment:
What's the benefit of this change compared to the original approach?

Reply:
I am trying to install a more recent version of Docker so we can use  DOCKER_BUILDKIT.